### ISSUE TITLE
HPI Generator The Way It Works Now

### ISSUE THREAD

**Issue by Brahmanathaswami** [Himalayan-Academy/HPI_Generator#1]
1) Google Doc - Red Zone
- Admin/Editor gets email or notification of a good article
- He goes the web site
- He copies to a GOOGLE doc
- Pastes the article, title and the URL  under the RED zone (at the bottom)

2) Volunteer
- Open Google Doc
- add delimiter "|" the  front
- summarizes or truncates the articles
- move it to the GREEN zone (at the top of the documentO

3) Admin/Editor
- Opens a previous HPI the BBdit.
- pastes the next artical
- convert to ASCII (dumbs down quote, m-dashes etc)
- marks sure the delimiter is there

4) Admin/Editor
- Open HPI Generator (LIvecode App)
- paste in the articles
- gets the quote of the daily

5) Uploads to XOOPs

6) Log it via web to XOOPS/Xnews to very articles.
- occasionally there are glitche the JSON Api it block the load
- admin editor tries to fix in at "home" (in BBEdit)
- Re-uploads
- confirms articles are good 
- Can see the articles the home pages of HToday site.
- signs off

7) The RSS feeds now has new articles

8) Next day around midnight, Feedblitz reads the RSS,  see new articlies and generation the email list.

---------------
Basically what have done is a 'blogging app" we did before they existed. Today, the whole thing wold be done in word press)

attached are;

1) Livecode stack
2) the new format for the new letter as an html template
[HPI_Generator.livecode.zip](https://github.com/Himalayan-Academy/HPI_Generator/files/2725991/HPI_Generator.livecode.zip)
[hpi_design-and-template.zip](https://github.com/Himalayan-Academy/HPI_Generator/files/2725993/hpi_design-and-template.zip)

**Comment by Brahmanathaswami** [comment]
@Arumugaswami check this out
